styles.css

```diff
@@ -1142,6 +1282,12 @@ textarea:focus {
     margin-top: 84px;
   }
 
+  /* .image-container {
+    width: 311px;
+    height: 220px;
+    top: 156px;
+  } */
+
```

Comment:
- [ ] [OPTIONAL]  I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile: